### PR TITLE
EDU-239 Fix prop types warnings

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-239

### CODE REVIEW

File: src/server/client-data-mapper.js [hunk 1]
```diff
@@ -3,12 +3,144 @@ import cloneDeep from '../utils/clone-deep.js';
...
+  const updatedOn = rawLesson.updatedOn && rawLesson.updatedOn.toISOString();
+  const schedule = rawLesson.schedule && mapLessonSchedule(rawLesson.schedule);
+
+  return {
+    ...rawLesson,
+    createdOn,
+    updatedOn,
+    schedule
+  };
+};
```
Comment (ahelmberger) [inline src/server/client-data-mapper.js:143]: Is there a reason why you implemented all those function outside of the class?

Comment (adrianaluchian) [reply src/server/client-data-mapper.js:143]: Felt like the class was super convoluted 🤔 But I'm not very attached to the idea, I can place them inside the class if you dislike this.

Comment (adrianaluchian) [reply src/server/client-data-mapper.js]: Reconsidered, moved everything in the class 👍

Comment (ahelmberger) [reply src/server/client-data-mapper.js]: Ok, I felt the class was a little bit empty before :-)

**Review by ahelmberger — APPROVED** [review]

**Pull request merged**